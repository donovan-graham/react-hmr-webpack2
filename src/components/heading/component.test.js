import React from 'react';
import Heading from './component';
import renderer from 'react-test-renderer';

describe('Heading component', () => {
  it('renders "Snappy"', () => {
    const tree = renderer.create(<Heading>Snappy</Heading>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders "Snapper"', () => {
    const tree = renderer.create(<Heading>Snapper</Heading>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
