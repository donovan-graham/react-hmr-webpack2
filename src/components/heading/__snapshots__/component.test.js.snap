// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component renders "Snapper" 1`] = `
<h1
  className="fJDhHD"
>
  Snapper
</h1>
`;

exports[`Heading component renders "Snappy" 1`] = `
<h1
  className="fJDhHD"
>
  Snappy
</h1>
`;
